website/style.css

```diff
@@ -1,74 +1,86 @@
-
 body {
-  color: #111;
+  color: #444;
   background: #f0f0f0;
-  margin: 1em;
-  font-family: Arial;
-  font-size: 20px;
+  padding: 32px 0;
+
+  line-height: 1.5;
+  font-family: sans-serif;
+
+  margin: 5ex 10ex;
+  max-width: 80ex;
 }
 
-main {
+#manual img {
+  width: 100%;
   max-width: 800px;
-  margin: 0px auto;
 }
-svg {
-  margin: 0px auto;
-  width: 100%;
-  height: 300px;
+
+
+h1, h2, h3, h4 {
+  font-weight: normal;
+  margin-bottom: 0;
 }
 
-a {
-  color: #333;
+h1, h2, h3, h4, p, table {
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+table {
+  border-collapse: collapse;
 }
-a:hover {
-  color: #488;
+td, th {
+  font-weight: normal;
+  text-align: center;
+  border: 1px dotted #bbb;
+  padding: 4px;
 }
 
-a.badge {
-  display: inline-block;
-  border-radius: 2px;
-  height: 20px;
-  transition: .3s box-shadow;
+svg {
+  margin: 0px;
+  width: 100%;
+  height: 300px;
 }
-a.badge:hover {
-  box-shadow: 0 0 5px 0 #333;
+
+a {
+  color: #247ee9;
```

Comment:
Can you change it to `#106ad5` to increase contrast and comply with [WCAG](https://www.w3.org/TR/WCAG20-TECHS/G18), please?
Text in the element `html > body > main > table > tbody > tr:nth-child(2) > th > a` has a contrast ratio of 3.52:1, while expected a contrast ratio of at least 4.5:1.

And also create a new rule for `pre a` with `color`: `#001631` for the same reason?
Text in the element `html > body > main > pre:nth-child(5) > a` has a contrast ratio of 1:1 (approximately, I suppose), while expected a contrast ratio of at least 4.5:1.

Reply:
done